bin/lux

```diff
@@ -32,6 +33,14 @@ function inLuxProject() {
   }
 }
 
+function commandNotFound(cmd) {
+  console.log();
+  console.log(`  ${red(cmd)} isn't a valid command`);
```

Comment:
I'm totally throwing stones in a glass house here but what do you think about changing the copy to following?

> ${red(cmd)} is not a valid command